codemon/CodemonHelp.py

```diff
@@ -2,28 +2,14 @@
 import os
 
 def showHelp():
-  os.system('cls sys')
   print(colored.green("           ---CODEMON---          "))
   print("A CLI tool for competitive coders")
   print("Get faster and more accurate during a coding contest.")

```

Comment:
You missed this line, please remove it.

Reply:
Edit the "A CLI tool for competitive coders" and change it to "A CLI tool to ace competitive programming contests"